framework/extensions/internal/extensionsconfiguration.cpp

```
@@ -144,29 +137,15 @@ std::map<muse::Uri, Manifest::Config> ExtensionsConfiguration::manifestConfigs()
             return {};
         }
 
-        std::map<Uri, Manifest::Config> result;
+        std::map<Uri, ExtensionConfig> result;
         JsonArray arr = doc.rootArray();
         for (size_t i = 0; i < arr.size(); ++i) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)